_includes/presentations.html

```diff
@@ -6,6 +6,7 @@ <h2>Presentations</h2>
                 <hr class="title-line check-calendar">
             </div>
             <ul class="presentations">
+                <li><a href="https://www.experoinc.com/online-seminar/janusgraph-online-meetup-3">JanusGraph Online Meetup</a>, Florian Grieskamp, Bruno Berisso, & Ted Wilmes - 2020.10.07</li>
```

Comment:
Probably it should be: https://www.experoinc.com/online-seminar/janusgraph-user-group-meet-up

The following link doesn't work for me: https://www.experoinc.com/online-seminar/janusgraph-online-meetup-3

Reply:
Good catch! Updated...